components/feature/prompts/src/main/java/mozilla/components/feature/prompts/dialog/AutofillEditText.kt

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package mozilla.components.feature.prompts.dialog
+
+import android.content.Context
+import android.os.Build
+import android.util.AttributeSet
+import android.view.ViewStructure
+import androidx.appcompat.widget.AppCompatEditText
+
+internal class AutofillEditText : AppCompatEditText {
```

Comment:
A class comment here would be useful, explaining that this exists to make sure we call `setWebDomain` on the view structure.

Reply:
Agreed.  I'll add the comment.  Thanks,